Dockerfile

```diff
@@ -6,7 +6,7 @@ VOLUME /config
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-RUN pip3 install --no-cache-dir colorlog cython
+RUN pip3 install --no-cache-dir colorlog cython mysqlclient psycopg2 uvloop
```

Comment:
I don't know about these dependencies, so maybe they can just move to the requirements.txt file?